engine/engine/src/engine_service.cpp

```diff
@@ -453,6 +467,11 @@ namespace dmEngineService
             upnp_params.m_Userdata = this;
             dmWebServer::AddHandler(web_server, "/upnp", &upnp_params);
 
+            dmWebServer::HandlerParams state_params;
+            state_params.m_Handler = StateHandler;
+            state_params.m_Userdata = this;
+            dmWebServer::AddHandler(web_server, "/state", &state_params);
```

Comment:
New endpoint `state`, because info provides static data we have even before engine initialization, but I need information we have only after the engine initialization.